game.rb

```diff
@@ -0,0 +1,128 @@
+require_relative 'dictionary'
+
+class Game
+  attr_reader :word, :guess
+
+  def initialize(word = nil)
+    @dictionary = Dictionary.new.trim_dictionary
+    @word = word || @dictionary.sample(1)[0].split('')
+    @used_letters = []
```

Comment:
Used letters and unused letters really represent the same value, no? Do you need both of these variables? Can `used` variables be a method calculated based on `unused_variables`, if it's really-really needed ?